tests/Transports.AspNetCore.Tests/NewtonsoftJsonTests.cs

```diff
@@ -41,64 +42,61 @@ public async Task Decodes_BigInteger()
             var request = @"{""variables"":{""a"":1234567890123456789012345678901234567890}}";
             var ret = await Deserialize(request);
             var bi = BigInteger.Parse("1234567890123456789012345678901234567890");
-            ret.Single.Inputs["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
+            ret.Single().Variables["a"].ShouldBeOfType<BigInteger>().ShouldBe(bi);
         }
 
         [Fact]
-        public async Task Dates_Should_Parse_As_Text()
+        public async Task Dates_Should_Parse_As_Dates()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            ret.Single().Variables["date"].ShouldBeOfType<DateTime>().ShouldBe(new DateTimeOffset(2015, 12, 22, 10, 10, 10, TimeSpan.FromHours(3)).LocalDateTime);
         }
 
         [Fact]
         public async Task Extensions_Null_When_Not_Provided()
         {
             var ret = await Deserialize(@"{""variables"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Extensions.ShouldBeNull();
+            ret.Single().Extensions.ShouldBeNull();
         }
 
         [Fact]
-        public async Task Name_Matching_Not_Case_Sensitive()
+        public async Task Name_Matching_Is_Case_Sensitive()
         {
-            var ret = await Deserialize(@"{""VARIABLES"":{""date"":""2015-12-22T10:10:10+03:00""}}");
-            ret.Single.Inputs["date"].ShouldBeOfType<string>().ShouldBe("2015-12-22T10:10:10+03:00");
+            var exception = await Should.ThrowAsync<Newtonsoft.Json.JsonException>(()
+                => Deserialize(@"{""VARIABLES"":{""date"":""2015-12-22T10:10:10+03:00""}}"));
+            exception.Message.ShouldBe("Exception of type 'Newtonsoft.Json.JsonException' was thrown.");
```

Comment:
Let's change behavior of serializer to skip unknown properties.

Reply:
This error tells about nothing.

Reply:
That would be a change in the main project.

Reply:
I know. Thanks for PR, reviewed.